src/local.js

```diff
@@ -46,6 +47,24 @@ function cachedToken(scope) {
 		});
 }
 
+function adjustClockSkew(res) {
+	var dateHeader = res.headers && res.headers.date;
+	if (!dateHeader) {
+		return;
+	}
+
+	var serverTime = new Date(dateHeader).getTime();
+	if (serverTime !== serverTime) {
```

Comment:
If the date string wasn't parse-able, then `getTime()` will return `NaN`, and `NaN !== NaN`